code/modules/surgery/bodyparts/parts.dm

```diff
@@ -4,8 +4,11 @@
 	desc = "It's impolite to stare at a person's chest."
 	icon_state = "default_human_chest"
 	max_damage = 200
+	///The bodytype required to attach to this chest
+	var/acceptable_bodytype = BODYTYPE_HUMANOID
```

Comment:
New vars go at the bottom of the declaration, instead of in the middle of the overrides